internal/manifests/collector/container.go

```diff
@@ -126,6 +142,22 @@ func getVolumeMounts(os string) corev1.VolumeMount {
 	return volumeMount
 }
 
+func getPrometheusVolumeMounts(os string) corev1.VolumeMount {
+	var volumeMount corev1.VolumeMount
+	if os == "windows" {
+		volumeMount = corev1.VolumeMount{
+			Name:      naming.PrometheusConfigMapVolume(),
+			MountPath: "C:\\Program Files\\Amazon\\AmazonCloudWatchAgent\\prometheusconfig",
```

Comment:
we are supporting prometheus for windows nodes too?

Reply:
Yeah.